.gitignore

```diff
@@ -0,0 +1,2 @@
+.*.swp
+.DS_Store
```

Comment:
No need for this file?

Reply:
It is nice to have for vi users and for macOS users to avoid committing unnecessary files. Is there any harm to it?

Reply:
OK, I just don't need it, so wasn't sure.